Packs/CommonScripts/Scripts/CopyNotesToIncident/CopyNotesToIncident.py

```diff
@@ -8,6 +8,11 @@
 ''' STANDALONE FUNCTION '''
 
 
+def remove_id_and_version_from_entry(entry):
+    del entry['ID']
+    del entry['Version']
```

Comment:
use `pop()` with `None` if the keys don't exist
```suggestion
    entry.pop('ID', None)
    entry.pop('Version', None)
```